libs/common/src/abstractions/devices/devices-api.service.abstraction.ts

```diff
@@ -0,0 +1,24 @@
+import { DeviceResponse } from "./responses/device.response";
+
+export abstract class DevicesApiServiceAbstraction {
+  getKnownDevice: (email: string, deviceIdentifier: string) => Promise<boolean>;
+
+  /**
+   * Get device by identifier (device identifier is client generated id - not device id in DB)
+   */
```

Comment:
Suggestion: You could use jsdoc for the parameter here. We also normally would put these on the implementation.

```suggestion
 /**
 * Get device by identifier
 * @param deviceIdentifier - client generated id (not device id in DB)
 */
```